src/js/ads.js

```diff
@@ -296,11 +305,13 @@ class VideoAds {
 	}
 
 	contentPauseRequestHandler() {
+		this.adsCompleted = false;
```

Comment:
This looks like it should be possible - appears Alberto set up some ads library mocking, so I can least try extending those events 👍